frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditComponentIdRow/EditComponentIdRow.module.css

```diff
@@ -11,6 +11,7 @@
   overflow: hidden;
   white-space: nowrap;
   text-overflow: ellipsis;
+  max-width: 245px;
```

Comment:
Hm, I dont think we should use fixed sizes like this 🤔

Reply:
fixed